specification/dartLangSpec.tex

```diff
@@ -14081,6 +14084,23 @@ \subsection{Property Extraction}
 (\ref{superGetterAccessAndMethodClosurization}).
 \EndCase
 
+\LMHash{}%
+\Case{Implicit}
+Let $e$ be an expression whose static type is
+an interface type that has a method named \CALL.
+In the case where the context type for $e$ is a function type,
```

Comment:
&mapsto; a function type or the type \FUNCTION,

Reply:
I left out `\FUNCTION` because we don't have that for the invocation, https://github.com/dart-lang/language/blob/123bad8162497c4fd54ef50fac186f8ed88773a1/specification/dartLangSpec.tex#L12781.

But I can see that the implementations (a couple of them, anyway) already support type `Function` as well, so we could also include it in both cases.

Reply:
Ah, by the way, this is not the receiver type, it is the context type, so there is no contradiction in using the type `Function` in the context as a request for a `.call`. Done!